src/modules/roc_pipeline/receiver_session_group.h

```diff
@@ -66,6 +66,10 @@ class ReceiverSessionGroup : public core::NonCopyable<> {
     const ReceiverConfig& receiver_config_;
 
     core::List<ReceiverSession> sessions_;
+
+    core::List<packet::PacketPtr> packet_ring_buffer_;
```

Comment:
Let's reflect the purpose of the ring buffer in its name. We could call it something like dropped_packets_, recently_dropped_packets, etc.